src/components/movie/MovieInlinePreview.tsx

```diff
@@ -38,7 +38,7 @@ class MovieInlinePreview extends React.Component<Props> {
     const { movie } = this.props;
     const { title, year, vote_average, poster_path } = movie;
     return (
-      <TouchableHighlightView scaleFactor={0.98} contentStyle={styles.container} onPress={this.onInlinePreviewPress} accessibilityLabel={title.replace(' ', '_')}  >
+      <TouchableHighlightView scaleFactor={0.98} contentStyle={styles.container} onPress={this.onInlinePreviewPress} accessibilityLabel={title.replace(/ /g, '_')}  >
```

Comment:
You can do this as well
```suggestion
      <TouchableHighlightView scaleFactor={0.98} contentStyle={styles.container} onPress={this.onInlinePreviewPress} accessibilityLabel={title.replaceAll(' ', '_')}  >
```

Reply:
Apparently RN does not support "replaceAll" function. We leave it as it is.